com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -998,6 +997,18 @@ define([
             function getContractData(contractId) {
               return me.contractsData[contractId];
             }
+
+            /**
+             * Parse dates so they can be correctly read by server.
+             *
+             * @param {string|Date} date
+             * @returns {string|null}
+             */
+            function convertDateToServerFormat(date) {
```

Comment:
Do you think this code may be useful in different CiviHR angular apps? 
Maybe it'd be good to have it in common dependency...